code/game/objects/items/weapons/tools/weldingtool.dm

```diff
@@ -68,6 +68,11 @@
 		if(!S || S.robotic < ORGAN_ROBOT || S.open == 3)
 			return ..()
 
+		//VOREStation Add - No welding nanoform limbs
+		if(S.robotic > ORGAN_LIFELIKE)
```

Comment:
This will include vey-med, zeng-hu & DSI limbs - ORGAN_NANOFORM should explicitly cover nanoform limbs afaik -- just saw the defines [here](https://github.com/VOREStation/VOREStation/blob/775d175b1146796f81b6a9673f222182cf2ef495/code/__defines/damage_organs.dm#L54) the code handling making it lifelike [here](https://github.com/VOREStation/VOREStation/blob/4f4a29b5fc264fab42eaf769e9df701b240293a1/code/modules/organs/organ_external.dm#L1120) and the actual limbs [here](https://github.com/VOREStation/VOREStation/blob/f60a39dd4d3504e217e190a97f63a9fb12be1396/code/modules/organs/robolimbs.dm) and [here](https://github.com/VOREStation/VOREStation/blob/f60a39dd4d3504e217e190a97f63a9fb12be1396/code/modules/organs/robolimbs_vr.dm)

Reply:
No, because S.robotic on those will be ORGAN_LIFELIKE. The test is *greater-than* ORGAN_LIFELIKE, not *greater-than-or-equal-to*